self-api/src/main/java/com/selfxdsd/api/User.java

```diff
@@ -66,4 +66,11 @@
      * @return Contributor or null, if this User is not one.
      */
     Contributor asContributor();
+
+    /**
+     * Returns the Admin if this User is one.
+     * @return Admin or null, if this User is not one.
+     * @todo: implement asAdmin method in all successors.
```

Comment:
@nmonokov I think this todo can have just 30min estimation, since it's quite simple.

Reply:
and what ordinal number should i use?